rules-java/tests/src/test/java/org/jboss/windup/rules/java/JavaHintsClassificationsTest.java

```diff
@@ -235,7 +259,13 @@ public void perform(GraphRewrite event, EvaluationContext context, JavaTypeRefer
                              .withEffort(8)
                     .and(addTypeRefToList))
             )
-            .where("suffix").matches("\\d");
+            .where("suffix").matches("\\d")
+                .addRule()
```

Comment:
Yeah, just bad formatting. The auto-formatter being off for these lines is kind of annoying.